README.md

```diff
@@ -11,6 +11,27 @@
 Documentation for the ORAS CLI is located on
 the project website: [oras.land/cli](https://oras.land/cli/)
 
+## Development Environment Setup
+
+You will need `git`, `go` and `make` installed on your system to start development.
+Once you have the basic tools installed, fork the repository on GitHub and clone it locally:
+```
+MY_GITHUB_NAME="my-name"
+git clone git@github.com:${MY_GITHUB_NAME}/oras.git
+cd oras
+```
+To compile and run the tests, just run the `make` command.
+If you want to create an executable, run `make build` which will build executables for several different operating systems and architectures.
+The `uname -a` command should give you some hints on which one to use:
+```
+% uname -a
+```
+Darwin 3c22fb4124 21.6.0 Darwin Kernel Version 21.6.0: Mon Dec 29 20:44:01 PST 2022; root:xnu-8020.240.18~2/RELEASE_X86_64 x86_64
```

Comment:
Do you want the output to into the code block. The readme currently renders without line breaks for Generally.

Reply:
fixed